src/components/content/inventory/InventoryList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {useInventoryList} from '../../../api/InventoryService';
```

Comment:
Reorder the imports so all relative imports are together

Reply:
they are, I think you mixed up `Inventory` and `Ingredient`